test/expectations/expectations.go

```diff
@@ -0,0 +1,63 @@
+package expectations
```

Comment:
Since this basically tests the whole generation, this should be in the package "test/generation" or "test/gen". "Expectations" does not describe what we are doing here.